oc/InterviewCdoe/InterviewCodeTest/InterviewCodeTest.m

```diff
@@ -0,0 +1,74 @@
+//
+//  InterviewCodeTest.m
+//  InterviewCodeTest
+//
+//  Created by Joe on 2022/6/16.
+//
+
+#import <XCTest/XCTest.h>
+#import "MyGreeterClient.h"
+
+@interface InterviewCodeTest : XCTestCase
+
+@property (nonatomic, strong) MyGreeterClient *testClient;
+
+@end
+
+@implementation InterviewCodeTest
+
+- (void)setUp {
+    [super setUp];
+    NSLog(@"setUp");
+    self.testClient = [[MyGreeterClient alloc] init];
+}
+
+- (void)tearDown {
+    [super tearDown];
+    NSLog(@"tearDown");
+    self.testClient = nil;
+}
+
+- (void)testInitClient {
+    XCTAssertTrue([self.testClient isMemberOfClass:[MyGreeterClient class]], @"create a instance of MyGreeterClient failed");
+}
+
+/* You could uncomment codes depend on your real time. */
```

Comment:
One thing you must satisfy is to make sure all tests are passing.

Reply:
I guess it didn't pass the 'testHelloNow' case . Please comment line 37 , uncomment the line 38, and it will work. The reason is that this case depends on the current time, as the line 35 shows ~~

Reply:
It seems that you missed the point for writing test cases.
The tricky thing about this question, is to make it work all the times.
You need figure out how to make it.

Reply:
Thank you for your correction~~

Reply:
Any updates? @YNBand 
Would you please close this PR if no further update?